support/build/_docker/heroku-22.Dockerfile

```diff
@@ -5,14 +5,19 @@ ENV WORKSPACE_DIR=/app/support/build
 ENV PATH=/app/support/build/_util:$PATH
 ENV S3_BUCKET=lang-php
 ENV S3_PREFIX=dist-heroku-22-develop/
-ENV S3_REGION=s3.us-east-1
+ENV S3_REGION=us-east-1
 ENV STACK=heroku-22
```

Comment:
(optional, and potentially for another PR anyway) Docker build args could reduce the duplicate of Dockerfiles here - ie: single `Dockerfile` that takes a `STACK_VERSION` arg, like:
https://github.com/heroku/heroku-buildpack-python/blob/ce39d0cc33c29c2583ff52fe2536e48908b9a0b8/builds/Dockerfile#L1-L5

Reply:
Yup, these are great, improvement for a future iteration :)